cli/src/test/unlock.test.ts

```diff
@@ -0,0 +1,69 @@
+import anyTest, { TestInterface } from 'ava';
+import * as td from 'testdouble';
+import { Context, addFake } from './fake_lycan';
+import { success } from 'specshell';
+import { Application } from '@binaris/spice-node-client/interfaces';
+import { UnauthorizedError, NotFoundError } from '@binaris/spice-koa-server/interfaces';
+
+const test = anyTest as TestInterface<Context>;
+
+addFake(test);
+
+const makeApp = (overrides?: Partial<Application>): Application => ({
+  accountId: '127001',
+  createdAt: new Date('1999-12-31T23:59:59.999Z'),
+  updatedAt: new Date('1999-12-31T23:59:59.999Z'),
+  locked: true,
+  lockReason: 'my-lock-app',
+  id: 'abc',
+  name: 'fluffy-pancake-66',
+  environments: [
+    {
+      name: 'default',
+      domains: [
+        {
+          type: 'subdomain',
+          name: 'a.b.c',
+        },
+      ],
+    },
+  ],
+  ...overrides,
+});
+
+const anything = td.matchers.anything();
+
+test.beforeEach(async (t) => {
+  t.assert(success(await t.context.shell.run(`cd ${t.context.projectDir}`)));
+  t.log(t.context.projectDir);
+});
+
+test('application successfully unlocked', async (t) => {
```

Comment:
Same as above, please give the test meaningful names so I know what you're testing.
Once you do that, I'll review the test code.